administrator/components/com_cache/Controller/DisplayController.php

```diff
@@ -65,7 +66,7 @@ public function getQuickiconContent()
 		$result = [];
 
 		$result['amount'] = $mb;
-		$result['sronly'] = Text::sprintf('COM_CACHE_QUICKICON_SRONLY', $mb);
+		$result['sronly'] = Text::sprintf('COM_CACHE_N_QUICKICON_SRONLY', $mb);
```

Comment:
You do not need the `_N_' here as it not a plural form.
Therefore it also has to be modified in the Cache DisplayController.

Reply:
Also:
I suggest to modify to not use `$mb` but only `$bytes`, as we would not get `MB` when we have plain numbers, i.e. 1, 2, etc.
Suggestion
```
		// Number bytes are returned in format xxx.xx MB
		$bytes = HTMLHelper::_('number.bytes', $size, 'MB', 1);

		$result = [];

		$result['amount'] = $bytes;
		$result['sronly'] = Text::sprintf('COM_CACHE_QUICKICON_SRONLY', $bytes);
```

Reply:
Resolved wit a new commit with better sronly message.